eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -145,6 +145,24 @@ jobs:
         $(CommonMSBuildArgs)
         $(OfficialBuildArg)
 
+  - ${{ if eq(parameters.osGroup, 'Android') }}:
+
+    - name: CommonMSBuildArgs
+      value: >-
+        /p:PortableBuild=true
+        /p:SkipTests=$(SkipTests)
+
+    - name: BaseJobBuildCommand
+      value: >-
+        $(Build.SourcesDirectory)/installer.sh --restore --build --ci --test
+        -configuration $(_BuildConfig)
+        -os ${{ parameters.osGroup }}
+        -arch ${{ parameters.archType }}
+        /p:StripSymbols=true
+        $(LiveOverridePathArgs)
+        $(CommonMSBuildArgs)
+        $(OfficialBuildArg)
```

Comment:
What's different here vs. the iOS case?

Reply:
Right, since theres no difference, combined the two.